addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -244,7 +244,8 @@ def _get_operator(self, previous_operator_id=None, lang=None, country_id=None):
         :return : user
         :rtype : res.users
         """
-        if not self.available_operator_ids:
+        final_operators = operators | self.available_operator_ids
```

Comment:
I'm checking with the PO but I don't think we want to fallback to `available_operator_ids` if sales person are given as param, just consider the step as unsuccessful and keep going with the chatbot if none of them matches.